src/pages/Announcement/Announcement.jsx

```diff
@@ -0,0 +1,7 @@
+import NewAnnouncement from '../../components/NewAnnouncement/NewAnnouncement.jsx';
+
+const Announcement = () => {
```

Comment:
it's unusable wrapper. Let's make without it

Reply:
delete & use componenent NewPost as element in the route